src/qml/main.qml

```diff
@@ -66,8 +73,35 @@ CutieWindow {
                 icon.height: width / 2
                 icon.width: height / 2
                 background: null
-                onClicked:
-                    compositor.execApp(model["Desktop Entry/Exec"]);
+
+                onPressed: {
+                    longPress = false
+                    longPressTimer.start()
+                }
+
+                onReleased: {
+                    longPressTimer.stop()
+                    if (!longPress)
+                        compositor.execApp(model["Desktop Entry/Exec"])
+                }
+            }
+
+            CutieMenu {
+                id: menu
+                width: window.width * 2 / 3
+                CutieMenuItem {
+                    text: qsTr("Add to favorites")
+                    onTriggered: {
+                        saveFavoriteItem(model["Desktop Entry/Name"], model["Desktop Entry/Icon"], model["Desktop Entry/Exec"]);
+                    }
+                }
+
+                CutieMenuItem {
```

Comment:
This should really be only on the actual favourites bar.

Reply:
Got it